pkg/api/server/register_play.go

```diff
@@ -46,6 +46,10 @@ func (s *APIServer) registerPlayHandlers(r *mux.Router) error {
 	//    description: Static MACs used for the pods.
 	//    items:
 	//      type: string
+	//  - in: query
```

Comment:
This is not RESTful, should be implemented as a DELETE method on a resource not a query parameter.

Reply:
seems wasteful to add a new endpoint for something like this ... see what others say and  ill decide ....

Reply:
I agree with both of you :angel:

Reply:
ok, lets do it by the book.